### PR TITLE
Changelog for v1.3.0

### PR DESCRIPTION
**Pull request by Lebedinsky**

### CODE REVIEW

**Review by LukasPaczos — APPROVED** [review]
:ship:

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 1.3.0 - December 17, 2020
+Visit our [1.0.0 migration guide](https://github.com/mapbox/mapbox-navigation-android/wiki/1.0-Navigation-SDK-Migration-Guide) and the [documentation pages](http://docs.mapbox.com/android/navigation/overview) for more information.
+
+### Changelog
+#### Features
+* Added `onFeedbackFlowFinished` method to `FeedbackFlowListener`. [3850](https://github.com/mapbox/mapbox-navigation-android/pull/3850)
+* Exposed an option for the user to provide an arrival experience feedback when the navigation session is finished. See `NavigationFeedbackOptions` for `NavigationView` integration and `FeedbackArrivalFragment` for standalone. [3758](https://github.com/mapbox/mapbox-navigation-android/pull/3758)
```
Comment (Guardiola31337) [inline CHANGELOG.md:11]: Is this duplicated? https://github.com/mapbox/mapbox-navigation-android/pull/3870/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR16

Comment (Lebedinsky) [reply CHANGELOG.md:11]: It was one PR with two changelog lines, so I decide to add both of them. Should I remove one of them?

Comment (Guardiola31337) [reply CHANGELOG.md:11]: Yeah maybe we can compact it into one entry. What do you think @LukasPaczos?

Comment (LukasPaczos) [reply CHANGELOG.md:11]: I'm fine leaving this as is since detailed feedback was already available and we changed it, while arrival is a new feature.

**Pull request merged**